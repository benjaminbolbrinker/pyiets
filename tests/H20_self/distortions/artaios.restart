/home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode0_1 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/sp /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode2_1 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode1_1 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode1_0 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode2_0 /home/bolbrinker/Coding/pyiets/tests/H20_self/distortions/mode0_0 