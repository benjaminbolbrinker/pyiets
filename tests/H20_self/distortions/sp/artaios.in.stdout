 ===========================================
 =             THIS IS ARTAIOS             =
 =               Version 2.0               =
 =                                         =
 =  C. Herrmann, L. Gross, T. Steenbock,   =
 =  M. Deffner, A. Voigt & G. C Solomon    =
 =                                         =
 =  Hamburg University, University of      =
 =  Copenhagen & Northwestern University   =
 =                2009-2016                =
 ===========================================

 Reading leftbas
 Reading centralbas
 Reading rightbas
 Reading leftatoms
 Reading centralatoms
 Reading rightatoms

====================================
  Full transmission for alpha electrons
====================================
------------------------------------
 Largest interelectrode H and S elements
------------------------------------
    -31.9142665133    0.6880028948

energy; transmission:         -5.000000         0.188028
  printing greensmatrix.1 ...
 Artaios terminated normally
